packages/react-ui-components/src/Tabs/tabs.spec.tsx

```diff
@@ -17,6 +17,7 @@ describe('<Tabs/>', () => {
         theme: {
             'tabs': 'baseTabsClassName',
             'tabs__content': 'basTabsContentsClassName',
+            'tabs__panel': 'basTabsPanelsClassName',
```

Comment:
typo `basT`

Reply:
Sorry, copy pasted from the line above..